lib/core/security/tokenRepository.js

```diff
@@ -189,9 +189,16 @@ class TokenRepository extends Repository {
 
     const parsedExpiresIn = parseTimespan(expiresIn);
 
+    let maxTTL;
+    if (type === 'apiKey') {
+      maxTTL = global.kuzzle.config.security.apiKey.maxTTL;
+    } else {
+      maxTTL = global.kuzzle.config.security.jwt.maxTTL;
+    }
```

Comment:
(nitpicking) More concise way of writing this:

```suggestion
    const maxTTL = type === 'apiKey'
      ? global.kuzzle.config.security.apiKey.maxTTL
      : global.kuzzle.config.security.jwt.maxTTL;
```